port.h

```diff
@@ -116,7 +116,15 @@ SOFTWARE.
 #define M_PI 3.14159265359
 #endif
 
+#endif  // CURRENT_WINDOWS
+
+#ifdef CURRENT_APPLE
+#ifdef CURRENT_APPLE_ENABLE_NSLOG
+#define CURRENT_NSLOG(...) ::NSLog(__VA_ARGS__)
+#else
+void CURRENT_NSLOG(...) {}
```

Comment:
The user may say something like `if (blah) CURRENT_NSLOG(...); else { ... }`. This macro would translate it into `if (blah) {}; else { ... }`, which won't compile.

A global no-op function is fine, but `{}` is not it.